File: C#/백준/Bronze/30402. 감마선을 맞은 컴퓨터/감마선을 맞은 컴퓨터.cs
using System;

namespace q{
    class w{
        static void Main(){
            for (int i =0; i<15; i++){
                string a = Console.ReadLine();
                bool b1 = a.Contains("w");
                bool b2 = a.Contains("b");
                bool b3 = a.Contains("g");
                if (b1){
                    Console.WriteLine("chunbae");
                    break;
                }
                if(b2){
                    Console.WriteLine("nabi");
                    break;
                }
                if(b3){
                    Console.WriteLine("yeongcheol");
                    break;
                }
            }
        }
    }
}

File: Java/백준/Bronze/30402. 감마선을 맞은 컴퓨터/감마선을 맞은 컴퓨터.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        for (int i =0; i<15; i++){
            String a = in.nextLine();
            boolean b1 = a.contains("w");
            boolean b2 = a.contains("b");
            boolean b3 = a.contains("g");
            if(b1){
                System.out.println("chunbae");
                break;
            }
            if(b2){
                System.out.println("nabi");
                break;
            }
            if(b3){
                System.out.println("yeongcheol");
                break;
            }
        }
    }
}